include/real_time_tools/thread.hpp

```diff
@@ -117,7 +117,9 @@ namespace real_time_tools {
     bool block_memory_;
 
     /**
-     * @brief Optimization level of the CPU. Set to 0 to get proper real-time performance.
+     * @brief Maximum desired latency of the CPU in microsecons. Set to 0 to get proper real-time 
```

Comment:
Typo: microsecon[d]s

Maybe: "Set to 0 to get [best] real-time"? - A higher value could be proper as well.

Reply:
done